test/cql-pytest/test_group_by_limit.py

```diff
@@ -0,0 +1,64 @@
+#
+# Copyright (C) 2022-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+import asyncio
+import time
+import pytest
+import logging
+from cassandra.protocol import InvalidRequest, ReadFailure                         # type: ignore
+from cassandra.query import SimpleStatement                                        # type: ignore
+
+
+logger = logging.getLogger(__name__)
+
+@pytest.mark.asyncio
+async def test_add_index(cql):
+    assert cql is not None
+    cql.execute("create keyspace ks with replication = {'class': 'SimpleStrategy', 'replication_factor': 1};")
```

Comment:
Please switch to NetworkTopology and use named DC, we want to migrate away and do not test SimpleStrategy + replication_factor tag is most likely going to be abandoned when switching to tablets.

Reply:
You don't need to do ANY OF THAT. This test doesn't need its own keyspace and can use the existing keyspace. Please take a look at existing cql-pytest tests and read test/cql-pytest/README.md to get a feel for this test framework, which focuses on quick tests that not only don't need to create a cluster for each test, it almost never needs to create a keyspace and often not even a table for each test (and several test can share the same table).